mitodl/odl-video-service
# Fix embedded video height/width

#### What are the relevant tickets?
Fixes #539

#### What's this PR do?
- Makes the recommended changes to VideoJS styles on embed pages as described in the issue.

#### How should this be manually tested?
- No changes should be obvious on video detail pages or anywhere else besides embed pages.
- Videos on embed pages should not include either of the following `max-height` in the `.vjs-tech` style class:
  ```css
   max-height: calc(100vh - 200px);
   max-height: calc(100vh - 60px);
  ```
- Create this file and load it into your browser using `python -m http.server`:
index.html:
  ```html
  <html>
    <body style="background-color: blue;">
      <iframe style='border: 1px solid red;' src="http://localhost:8089/videos/<your_video_key>/embed/" width="554" height="336" frameborder="0" allowfullscreen></iframe>
    </body>
  </html>
  ```
- The video should take up the full width of the iframe, and the empty space below it within the iframe should be blue (the page background color) and not white.
